app/graphql/mutations/add_assets_to_consignment_submission.rb

```diff
@@ -2,13 +2,15 @@
 
 module Mutations
   class AddAssetsToConsignmentSubmission < Mutations::BaseMutation
-    argument :gemini_tokens, [String], required: true
+    argument :gemini_tokens, [String], required: false
     argument :submissionID, ID, required: false
     argument :external_submission_id, ID, required: false
     argument :sessionID, String, required: false
     argument :asset_type, String, required: false
     argument :filename, String, required: false
     argument :size, String, required: false
+    argument :s3_buckets, [String], required: false
+    argument :s3_paths, [String], required: false
```

Comment:
Thoughts on supporting an extra object type for an `UploadSource` to encapsulate the key+bucket pairs?

Thinking of something like:

```graphql
{
  addAssetToConsignmentSubmission(input: {
    source: { key: "KEY", bucket: "BUCKET" }
  }) {
    //
  }

  addAssetsToConsignmentSubmission(input: {
    sources: [{ key: "KEY", bucket: "BUCKET" }]
  }) {
    //
  }
}
```

Reply:
Also noticed that we tend not to encode "s3" into field names in Metaphysics. Probably in an effort not to lock the schema to any one provider. So might be worth choosing more generic names here as well.